packages/core/status/core-status-common/src/service_status.ts

```diff
@@ -39,6 +39,13 @@ export interface ServiceStatus<Meta extends Record<string, any> | unknown = unkn
   meta?: Meta;
 }
 
+/**
+ * Possible values for the ID of a {@link ServiceStatusLevel}
+ *
+ * @public
+ */
+export type ServiceStatusLevelId = 'available' | 'degraded' | 'unavailable' | 'critical';
```

Comment:
We actually didn't have a type for the possible status values. I added it, as it cleared some of the internal types.

Reply:
💯